docs/podman-run.1.md

```diff
@@ -260,7 +260,7 @@ You need to specify multi option commands in the form of a json string.
 
 Set environment variables
 
-This option allows you to specify arbitrary environment variables that are available for the process that will be launched inside of the container. If you specify a environment variable without a value, podman will check the host environment for a value or set the environment to "". If you specify a environment variable ending in --*--, podman will search the host environment for variables starting with the prefix and add them to the container.  If you want to add an environment variable with a ***** following it, then you need to set a value.
+This option allows arbitrary environment variables that are available for the process to be launched inside of the container.  If an environment variable is specified without a value, Podman will check the host environment for a value and set the variable only if it is set on the host.  If an environment variable ending in __*__ is specified, Podman will search the host environment for variables starting with the prefix and will add those variable to the container.  If an environment variable with a trailing ***** is specified, then a value must be supplied.
 
 See **Environment** note below for precedence.
```

Comment:
link (if possible) and add `and examples` at the end

Reply:
examples under **Environment** or in **--env**?